CDTDatastoreReplicationAcceptanceTests/CloudantReplicationBase.m

```diff
@@ -107,6 +111,26 @@ - (NSString*)createTemporaryDirectoryAndReturnPath
     return path;
 }
 
+/**
+ Get the IAM access token.  Used for CRUD helper methods.
+ */
+-(NSString *) getIAMBearerToken {
```

Comment:
I'm guessing there is no easy way to just call the method we have in the interceptor to do this instead of having this UNIRest based version of getting a token?

Reply:
If we change the currently private `getBearerToken` to public, then we can use the interceptor to do this:
https://github.com/cloudant/CDTDatastore/blob/0cb4502e09ce5a801c580d8112510ff7e8c6cda0/CDTDatastore/HTTP/CDTIAMSessionCookieInterceptor.m#L97

Reply:
Fair enough, I don't think it is worth exposing that for this purpose.